cli/commands/env.go

```diff
@@ -0,0 +1,33 @@
+package commands
```

Comment:
@vdice nope, it just means that they are inaccessible to code that's ouside of the `commands` package. the commands available to the user are specified in `cli/main.go`